test/pylib/scylla_cluster.py

```diff
@@ -419,12 +426,16 @@ async def start(self, api: ScyllaRESTAPIClient) -> None:
             if hasattr(self, "host_id") or await self.get_host_id(api):
                 cql_up_state = await self.cql_is_up()
                 if cql_up_state == CqlUpState.QUERIED:
+                    if expected_error is not None:
+                        raise RuntimeError(f"Server {self.server_id}, IP = {self.ip_addr}, started but was expected to fail with error [{expected_error}]")
```

Comment:
Please include the path to the log file in this message - like the message above does - to make it easier to debug a test failure.